packages/pds-ember/app/styles/pds/components/button/_config.scss

```diff
@@ -55,6 +55,7 @@ $paddingInline--nearIcon: scale.$lg--1; // 1rem (16px)
   height: $height;
   justify-content: center;
   padding: 0 $paddingInline;
+  white-space: nowrap;
```

Comment:
big pr for this little guy. : )

Reply:
well, sort of... Given that I had to add a story to demonstrate the change, I went ahead and updated the syntax to what is recommended for Storybook 6.